client/src/modules/Course/Styles/ReviewModal.module.css

```diff
@@ -119,6 +119,16 @@
   place-self: center;
 }
 
+.anonymouslabel {
+  display: flex;
+  font-size: 1em;
```

Comment:
For consistency with the codebase, it'd be best to use pixels for margin size and font size

Reply:
I also think that since it's in a flex-column already, you might not need bottom and top margins because it should just use the gap attribute - you can play around with decreasing the gap size in `.textcol` since you added a new text row.